### PR TITLE
Bump phel-lang to 0.1.0 and exclude composer.lock

### LINKED ISSUE #9: Update composer with release version
## 📚  Description

Use the `0.1.0` version for the `phel-lang` dependency instead of the `dev-master` one.

### PR DESCRIPTION
**Pull request by ChaseKnowlden**
## 📚 Description

Bumps phel-lang to 0.1.0, Closes #9 

## 🔖 Changes

- Update phel-lang to 0.1.0

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "license": "MIT",
   "require": {
     "ext-readline": "*",
-    "phel-lang/phel-lang": "dev-master"
+    "phel-lang/phel-lang": "0.1.0"
```
Comment (Chemaclass) [inline composer.json]: I would rather suggest `^0.1` 👌

**Review by Chemaclass — COMMENTED** [review]
I am not sure if we want to (git)ignore the `composer.lock` file in this template, but certainly, we wanted to update the composer dependency version. Anyway, it's a valid point.

What do you think, @jenshaase, about adding the `composer.lock` to `.gitignore`?

**Comment by jenshaase** [comment]
Yes I is a good idea to ignore it. Otherwise it will be committed somewhere else.

File: composer.json (hunk 1 above)
Comment (ChaseKnowlden) [reply composer.json]: I changed it already.

**Pull request merged**